tutorials/physics/interpolation/physics_interpolation_introduction.rst

```diff
@@ -0,0 +1,131 @@
+
+However, modern games are often no longer made for fixed hardware. You will often be planning to release on desktop computers, mobiles and more, all of which have huge variation in performance, as well as different monitor refresh rates. We need to come up with a better way of dealing with the problem.
+
+Adapt the tick rate?
+^^^^^^^^^^^^^^^^^^^^
+
+Instead of designing the game at a fixed physics tick rate, we could allow the tick rate to scale according to the end users hardware. We could for example use a fixed tick rate that works for that hardware, or even vary the duration of each physics tick to match a particular frame duration.
+
+This works but there is a problem - physics (*and game logic*, which is often also run in the ``_physics_process``) work best and most consistently when run at a **fixed**, pre-determined tick rate. If you attempt to run a racing game physics that has been designed for 60tps (ticks per second) at e.g. 10tps, the physics will behave completely differently. Controls may be less responsive, collisions / trajectories can be completely different. You may test your game thoroughly at 60tps, then find it breaks on end users machines when it runs at a different tick rate.
+
+This can make quality assurance difficult with hard to reproduce bugs, especially in AAA games where problems of this sort can be very costly.
+
+Lock the tick rate, but use interpolation to smooth frames in between physics ticks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This has become one of the most popular approaches to dealing with the problem, and is automatically supported by Godot (although is optional).
+
+We have established that the most desirable physics / game logic arrangement for consistency and predictability is a physics tick rate that is fixed at design time. The problem is the discrepancy between the physics position recorded, and where we "want" a physics object to be shown on a frame to give smooth motion.
+
+The answer turns out to be simple, but can be a little hard to get your head around at first.
+
+Instead of keeping track of just the current position of a physics object in the engine, we keep track of *both the current position of the object, and the previous position* on the previous physics tick.
+
+Why do we need the previous position *(in fact the entire transform, including rotation and scaling)*? Well by using a little maths magic we can use **interpolation** to calculate what the transform of the object would be between those two points, in our ideal world of smooth continuous movement.
```

Comment:
```suggestion
This has become one of the most popular approaches to dealing with the problem. It's supported by Godot 3.5 and later in 3D (although it's optional and disabled by default).

We have established that the most desirable physics/game logic arrangement for consistency and predictability is a physics tick rate that is fixed at design-time. The problem is the discrepancy between the physics position recorded, and where we "want" a physics object to be shown on a frame to give smooth motion.

The answer turns out to be simple, but can be a little hard to get your head around at first.

Instead of keeping track of just the current position of a physics object in the engine, we keep track of *both the current position of the object, and the previous position* on the previous physics tick.

Why do we need the previous position *(in fact the entire transform, including rotation and scaling)*? By using a little math magic, we can use **interpolation** to calculate what the transform of the object would be between those two points, in our ideal world of smooth continuous movement.
```